connector/oidc/oidc.go

```diff
@@ -259,11 +295,21 @@ func (c *oidcConnector) HandleCallback(s connector.Scopes, r *http.Request) (ide
 		}
 	}
 
+	cd := connectorData{
+		refreshToken: []byte(token.RefreshToken),
+	}
+
+	connData, err := json.Marshal(&cd)
```

Comment:
Hi just started poking into this PR because I am looking to use something similar, and because connectorData.refreshToken is not exported, it cannot be exported to JSON correct?

Reply:
That's a very very good point, thanks for spotting that! I've fixed it now 🙂